.devcontainer/devcontainer.json

```diff
@@ -35,6 +35,7 @@
         "marus25.cortex-debug",
         "ms-azuretools.vscode-docker",
         "msedge-dev.gnls",
+        "ms-vscode.cpptools",
```

Comment:
Please add cpptools to recommended extensions as well: https://github.com/project-chip/connectedhomeip/blob/master/.vscode/extensions.json

Reply:
@aalexanderr ?

Reply:
done, amended commit pushed